index.js

```diff
@@ -78,6 +83,196 @@ function loader (version) {
         return new Item(item.blockId, item.itemCount, item.itemDamage, item.nbtData)
       }
     }
+
+    static getEnchants (item) {
+      const normalize = (enchList) => enchList.map(ench => {
+        return { lvl: ench.lvl, name: mcData.enchantments[ench.id].name }
+      })
+      if (mcData.isOlderThan('1.13')) {
+        let itemEnch
+        if (item.name === 'enchanted_book' && item.nbt !== null) {
+          itemEnch = nbt.simplify(item.nbt).StoredEnchantments
+        } else if (item.nbt !== null) {
+          itemEnch = nbt.simplify(item.nbt).ench
+        } else {
+          itemEnch = []
+        }
+        return normalize(itemEnch)
+      }
+      const gg = nbt.simplify(item.nbt)
+      console.log(gg)
+      return 1
+    }
+
+    // denormalize converts normalized enchants back to 1.8 enchant format with {id, lvl}
+    static denormalize (enchantList) {
+      const findEnch = (name) => Object.entries(mcData.enchantments).map(x => x[1]).find(x => x.name === name)
+      return enchantList.map(ench => ({ lvl: ench.lvl, id: findEnch(ench.name).id }))
```

Comment:
Should this function also contain the de-simplification of nbt ? (Basically re-add the types) This would make it more useful.

Reply:
if it does that too, maybe I should rename it to toNbt? and make it take a version so then it can print to either

Reply:
added setEnchant